### ISSUE TITLE
Update documentation - MCP > URL Asset Import - Include text to inform users that Namespace must exist on AEM

### ISSUE THREAD

**Issue by ksurendra** [Adobe-Consulting-Services/acs-aem-commons#1519]
## Required Information

* [ ] AEM Version, including Service Packs, Cumulative Fix Packs, etc: AEM 6.3 CFP 2
* [ ] ACS AEM Commons Version: 3.18.0
* [ ] Reproducible on Latest? YES

## Background on the issue
If the spreadsheet has a column with a namespace prefix (like msft:fieldName), and if the namespace(say msft) does not exist on AEM; MCP > URL Asset Import will write it to AEM as **msft%3AcolumnName**. But if the namespace exists, then the data will be written as **msf:columnName** 

## Page location to include the above text(similar to):
On this page https://adobe-consulting-services.github.io/acs-aem-commons/features/mcp-tools/asset-ingestion/url-asset-ingestor/index.html under section **Note on working with cq:tags and other kinds of data types**

### Links
https://adobe-consulting-services.github.io/acs-aem-commons/features/mcp-tools/asset-ingestion/url-asset-ingestor/index.html

**Comment by badvision** [comment]
Although true, this would be the case if you were using the sling post servlet, etc.   It's kind of a general rule of thumb to check that stuff. ;)

**Comment by pahupe** [comment]
Correct. The requested documentation should be provided. Also, pointers to how to create namespaces via CND files should be added, since the nodetype admin UI is not available in AEMaaCS. I will see if I find links to documentation that explains the process.
